vision-and-language/clip/main.py

```diff
@@ -0,0 +1,24 @@
+import nnabla as nn
+import nnabla.functions as F
+from PIL import Image
+
```

Comment:
Currently your script runs on CPU. Add context argument to enable users to select which environment to run on.
Something like
```
    from nnabla.ext_utils import get_extension_context
    ctx = get_extension_context(
        args.context, device_id=args.device_id, type_config=args.type_config)
    nn.set_default_context(ctx)
```